From a3f76dccdef8b2968a1c9d6961585ae9162e9eb3 Mon Sep 17 00:00:00 2001
From: Justin Kim <justin.kim@collabora.com>
Date: Thu, 23 Nov 2017 20:03:57 +0900
Subject: [PATCH] channel.cpp: add mswsock.h for Win32

For cross-compiling, 'mswsock.h' is required to use
MSG_TRUNC flag.

Signed-off-by: Justin Kim <justin.kim@collabora.com>
---
 srtcore/channel.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/srtcore/channel.cpp b/srtcore/channel.cpp
index a15179b..07b0b96 100644
--- a/srtcore/channel.cpp
+++ b/srtcore/channel.cpp
@@ -75,6 +75,7 @@ modified by
 #else
    #include <winsock2.h>
    #include <ws2tcpip.h>
+   #include <mswsock.h>
 #endif
 
 #include <iostream>
-- 
2.7.4

